src/Umbraco.Core/ServerEvents/IEventSourceAuthorizer.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Security.Claims;
+
+namespace Umbraco.Cms.Core.ServerEvents;
+
+/// <summary>
+/// Authorizes a Claims principal to access an event source.
+/// </summary>
+public interface IEventSourceAuthorizer
+{
+    public IEnumerable<string> AuthorizedEventSources { get; }
```

Comment:
The code tells me that these are the eventSources for which this Authorizer should be run, not which eventSources have been authorized, might have to rethink the name if not reworked because of other comments.